templates/constraints/yml/templates/csharp/link-unfurling/teamsapp.local.yml.tpl.mustache

```diff
@@ -7,12 +7,34 @@ provision:
 
 {{#fileCreateOrUpdateJsonFile}} appsettings, BOT {{/fileCreateOrUpdateJsonFile}}
 
-{{#botFrameworkCreate}} {{/botFrameworkCreate}}
+{{#botFrameworkCreate}} m365extensions {{/botFrameworkCreate}}
```

Comment:
Just a double confirm, add this input then botFramework/create can generate the 'channel name' value, right?

Reply:
Yes, it will add m365extensions in the channels.